app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -47,6 +47,12 @@ def update
           selected_location_details: update_params.as_json,
           issuer: current_sp&.issuer,
         )
+
+        if document_capture_session
+          enrollment.update!(
+            doc_auth_result: document_capture_session.last_doc_auth_result,
+          )
```

Comment:
should we consider making 1 database call to update these attributes instead of 2 database calls back to back?